components/TField.vue

```diff
@@ -69,6 +72,10 @@ export default {
       type: String,
       default: '',
     },
+    error: {
+      type: String,
+      default: null,
```

Comment:
I would rather use empty string here to comply with type. AFAIK null is an object.

```suggestion
      default: '',
```